nowcasting_datamodel/models/forecast.py

```diff
@@ -327,10 +327,14 @@ def from_orm(cls, obj: ForecastValueSQL):
 
         # this is because from orm doesnt copy over '_' variables.
         # But we don't want to expose this in the API
+        default_value = 0.0
         if hasattr(obj, "adjust_mw"):
-            m._adjust_mw = obj.adjust_mw
+            adjust_mw = obj.adjust_mw
+            if not adjust_mw:
```

Comment:
should this be` if np.isnan(adjust_mw)`

Reply:
Maybe its the same thing, but what i suggested it perhaps more readable

Reply:
Good call, thanks.

Unfortunately `np.isnan` doesn't cover the case where `adjust_mw` gets set to `None` by Pydantic when it's completely missing, giving a `TypeError`. But I'm not sure if a `None` _can_ ever sneak in here in prod. Added an `np.isnan` check to cover all scenarios as well as the `None` check, but let me know if you'd prefer just leaving out the latter.

Reply:
Cool- that looks good - thank you. I prefer leaving it in - thanks